inst/rmd/h/navigation-1.1/codefolding-lua.css

```diff
@@ -0,0 +1,9 @@
+.chunk-details[open] > .chunk-summary::after {
+  content: "Hide";
+}
+.chunk-details[open] .chunk-summary-text {
+  display: none;
+}
+.chunk-summary {
+  text-align: right;
+}
```

Comment:
Just a thought:  What about being more specific in the selector ? 
 like `details.chunk-details` and `summary.chunk-summary` and `span.chunk-summary-text`. This would insure a higher specificity in the selector, less prone to conflict potentially.

Reply:
Alright.

Reply:
fixed by [b28dfa7](https://github.com/rstudio/rmarkdown/pull/2340/commits/b28dfa7491e8b19059cf04f55ed29b283abed483)